src/platform/assets/components/AssetGrid.vue

```
@@ -34,19 +36,17 @@
         :asset="asset"
         :interactive="true"
-        role="gridcell"
         @select="$emit('assetSelect', $event)"
       />
     </template>
   </div>
 </template>
 
 <script setup lang="ts">
-import { computed } from 'vue'
+import { cn } from '@comfyorg/tailwind-utils'
```

ESLint

[error] Unable to resolve path to module '@comfyorg/tailwind-utils'.

(import-x/no-unresolved)